tests/console/hostname.pm

```diff
@@ -40,6 +41,13 @@ sub run {
         assert_script_run "ip addr show br0 | grep UP";
         save_screenshot;
     }
+
+    if (sle_version_at_least('15')) {
+        record_soft_failure "bsc#981651";
```

Comment:
We should only record a soft-failure when you detected the error condition. I know that it might be tricky to do. I don't know if you can instruct local tools to force a *remote* DNS resolution. An alternative might be on the client side to try DNS resolution and if that fails then fall back to IP-address. You can then even try to recover by accessing the host over IP and restart the network remotely and then call `record_soft_failure`.

Also the referenced bug is against SLE 12 SP2 so I wonder why you actually need to do a workaround for SLE >= 15 ?

Reply:
Condition for issue detection should be better now. I would like to extend while condition to check that we are in multimachine job with supportserver with DNS role. @okurz Is it possible? If so, could you please give me some hint?